tests/cmds/test_cases.py

```diff
@@ -0,0 +1,74 @@
+from code42cli.main import cli
+
+
+def test_create_case_calls_create_with_expected_params(runner, cli_state):
+    runner.invoke(
+        cli, ["cases", "create", "TEST_CASE"], obj=cli_state,
+    )
+    cli_state.sdk.cases.create.assert_called_once_with(
+        "TEST_CASE", assignee=None, description=None, findings=None, subject=None
+    )
+
+
+def test_update_case_calls_update_with_expected_params(runner, cli_state):
+    runner.invoke(
+        cli, ["cases", "update", "1", "--name", "TEST_CASE2"], obj=cli_state,
+    )
+    cli_state.sdk.cases.update.assert_called_once_with(
+        1,
+        name="TEST_CASE2",
+        status=None,
+        assignee=None,
+        description=None,
+        findings=None,
+        subject=None,
+    )
+
+
+def test_list_calls_get_all_with_expected_params(runner, cli_state):
```

Comment:
None of these test are verifying output.  We should have some tests that verify some data is present in the show and list commands via `res.output` (if you set `res =` the runner invocation.

See other tests in the CLI as an example.

Reply:
Added new tests to cover that.